catalog/imaging/imaging-actionprovider-chip/src/test/java/org/codice/alliance/imaging/chip/actionprovider/TestImageChipActionProvider.java

```diff
@@ -64,8 +71,8 @@ public void testCanHandleInvalidMetacard() {
 
     @Test
     public void testCanHandleInvalidLocationOnMetacard() {
```

Comment:
Nit: the name of this test implies to me that you think it "can handle and invalid location". I'd change to something like `testDoesNotHandleInvalidLocation()`